Formula/pmd.rb

```diff
@@ -11,10 +11,35 @@ class Pmd < Formula
     sha256 "e5ed61e35b5c980b0e1bcf616bc2615f1c5310ad66ce8f4901ca225764e7e78c" => :mavericks
   end
 
+  depends_on :java => "1.8+"
   depends_on "maven" => :build
 
   def install
-    ENV.java_cache
+    ENV["_JAVA_OPTIONS"] = "-Duser.home=#{buildpath}/java_user_home"
```

Comment:
Could we maybe make a symlink back to the `.m2/repository` cache directory so that it's used? For bonus points: maybe this is the only thing we should be putting in the cache with `ENV.java_cache` at all CC @DomT4 ?